### ISSUE TITLE
TIFF

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#495]
@joshmoore @bugraoezdemir 

Does TIFF officially only have metadata for each plane (aka IFD), or is there also a place for some "overall metadata"?

**Comment by joshmoore** [comment]
If you mean non-OME-TIFF, I don't know. Googling I see https://stackoverflow.com/questions/1442218/multipart-tiff-and-exif-metadata but how _official_ that is is always a question.

**Comment by tischi** [comment]
@joshmoore and are you in OME-TIFF just treating the first IFD special or where do you put general metadata?

**Comment by joshmoore** [comment]
There's a `PlaneInfo` field in the XML

**Comment by tischi** [comment]
And where is the XML? Is it part of the first IFD?

**Comment by joshmoore** [comment]
Exactly, from https://docs.openmicroscopy.org/ome-model/6.3.1/ome-tiff/specification.html

> a string of OME-XML metadata embedded in the ImageDescription tag of the first IFD (Image File Directory) of each file. The XML string must be UTF-8 encoded.
